### ISSUE TITLE
`expiring-todo-comments` - more specific error message for missing conditions

### ISSUE THREAD

**Issue by mmkal** [sindresorhus/eslint-plugin-unicorn#744]
Rather than falling back to `no-warning-comments` when "useless" todos are found, could the rule explicitly report something like `TODO comment should have a condition or due date`?

When I first saw `Unexpected "todo" comment` I thought "oh, this is a rule which aggressively says 'todos are bad'", and I turned it off. Later, I spent a bunch of time implementing a worse version of `expiring-todo-comments`, which parsed ISO dates. Eventually I stumbled across the docs and realised how useful this rule really is.

Basically the problem is that it's piggy-backing off eslint's error message, which is too broad and doesn't really nudge the user into fixing the problem - and risks pushing them into deleting the todo or turning off the rule.
